openpype/hosts/houdini/startup/MainMenuCommon.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <mainMenu>
     <menuBar>
-        <subMenu id="openpype_menu">
+        <subMenu id="ayon_menu">
```

Comment:
I would keep names as openpype until it is separated to addon. Only visible labels should be affected.